.github/workflows/build.yaml

```diff
@@ -0,0 +1,113 @@
+name: "build"
+on:
+    push:
+        branches:
+            - "master"
+    pull_request: ~
+
+jobs:
+    coding-standards:
+        name: "Coding Standards"
+
+        runs-on: "ubuntu-latest"
+
+        strategy:
+            matrix:
+                php-version:
+                    - "7.3"
```

Comment:
i think we'll want to add 5.6 here as well since this plugin still can be used by older versions / other integrators still run this version

Reply:
Let's try :)